queries/examples/grammar.py

```diff
@@ -0,0 +1,86 @@
+"""
+
+    Greynir: Natural language processing for Icelandic
+
+    Example of a grammar query processor module.
+
+    Copyright (C) 2020 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+
+
+    This module is an example of a plug-in query response module
+    for the Greynir query subsystem. It handles grammar queries, i.e.
+    queries that require grammatical parsing of the query text.
+
+
+"""
+
+
+# Indicate that this module wants to handle parse trees for queries,
+# as opposed to simple literal text strings
+HANDLE_TREE = True
+
+# The context-free grammar for the queries recognized by this plug-in module
+GRAMMAR = """
+
```

Comment:
Might want to include the lemma stuff here with examples, i.e. the presence of the lemma "prufa" causes a response of "Þú gætir spurt til dæmis 'er þetta prufa'?"